### DISCUSSION TITLE
How to change q-table sorting icon

### DISCUSSION THREAD

**Discussion by shomalikhashayar** [quasarframework/quasar#16239] [General - Components / Directives / etc]
Hi guys, I use quasar q-table and I have a problem. I want change sort icon to material icons "arrow_drop_down" but can't find where should I change it.

![image](https://github.com/quasarframework/quasar/assets/118342117/71610086-49ca-41af-8c7a-da11f28dbc37)

**Comment by yusufkandemir** [comment]
https://quasar.dev/options/quasar-icon-sets#changing-a-specific-icon-dynamically
https://github.com/quasarframework/quasar/blob/a4bc2e42e1b5c52080913fe6d678339d20949b13/ui/icon-set/material-icons.mjs#L110C29-L110C29
In light of the sources, here is the solution:
```js
// App.vue
$q.iconSet.table.arrowUp = 'arrow_drop_up'
```
QTable uses arrow up form, then rotates it by 180 degrees when needed, that's why I used `arrow_drop_up` above.
